### PR TITLE
Integrate black

### LINKED ISSUE #19: Integrate black for python code formatting
https://github.com/psf/black

### PR DESCRIPTION
**Pull request by markrawls**
Closes #19 
Supercedes #2

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,15 +16,15 @@ requests = "^2.23.0"
...
 [tool.poetry.dev-dependencies]
 pytest = "^3.0"
-flake8 = "^3.7"
 autopep8 = "^1.5"
+black = "^19.10b0"
 
 [tool.poetry.scripts]
 zg = 'zygoat.cli:cli'
+[tool.black]
+line-length = 95
```
Comment (Ian-MacLeod) [inline pyproject.toml]: Why the change from 120->95? Just wondering, I'm not necessarily opposed to it.

Comment (kborer) [reply pyproject.toml]: let's stick with 120

Comment (markrawls) [reply pyproject.toml]: @Ian-MacLeod The docs make a rather compelling case for <100. Prettier's make the same arguments but with the heavily nested indentation of React that's frequently just not realistically doable.

@kborer It makes no difference to me, but I want to hear everyone else's opinions on it.

@cviveiro You seem to have opinions on this. Since it's an autoformatter and I never have to actually write code in that manner I don't really care - share your thoughts!

Comment (cviveiro) [reply pyproject.toml]: I prefer <100 as a general rule, I hate massive lines. But that's only if this is only working on new code, i.e. it won't get in the way of developing in the willing app. Which I think is the case.

Comment (markrawls) [reply pyproject.toml]: We could also run Black against the willing app if we so desired to hammer it into line, but that's your call.

**Comment by markrawls** [comment]
Added it to the pre-commit config
